packages/experiments/src/components/Chiclet/Chiclet.styles.ts

```diff
@@ -0,0 +1,132 @@
+import { memoizeFunction } from '../../Utilities';
+import {
+  ITheme,
+  concatStyleSets,
+  getTheme
+} from '../../Styling';
+import { IChicletCardStyles } from './ChicletCard.types';
+
+const ChicletCardTitleLineHeight = '21px';
+
+/* Actions */
+const msChicletCardActionsActionSize = '34px';
+const msChicletCardActionsHorizontalPadding = '12px';
+const msChicletCardActionsVerticalPadding = '2px';
+
+export const getClassNames = memoizeFunction((
```

Comment:
Please avoid using `memoizeFunction` and `getClassNames`. Instead, use `styled` helper and the `getStyles` prop to make things style-able. 

Follow the guidance here:

https://github.com/OfficeDev/office-ui-fabric-react/blob/master/ghdocs/BestPractices/Styling.md